rasa/cli/test.py

```diff
@@ -207,4 +216,4 @@ def test(args: argparse.Namespace):
     unpacked_model = get_model(model_path)
 
     test_core(args, unpacked_model)
-    test_nlu(args, unpacked_model)
+    test_nlu(args, model_path)
```

Comment:
I actually would prefer to pass the `unpacked_model` to both methods. Otherwise, we have some commands were you can provide a path to the unpacked model and other commands just take the `tar.gz` model files. To keep it consistent, I would prefer using the zipped file or the folder containing the zipped file as argument.

Reply:
Ok, this is currently not working as `rasa test core` does compare training if a models folder with multiple unpacked models is given. So will keep it the way it currently is. On a side note, compare training currently also does not work. 
My suggestion: Add a subparser for `rasa test core compare` to separate the two commands. This way we can also make sure that we always only expect `tar.gz` model files. What do you think? If you agree, I'll add it to the ticket.

Reply:
Mhm, although `rasa test core compare` would be more easier, it would also be less intuitive and a longer command as well. Could we not check if a file or a directory was passed and then decide whether to do regular or compare testing?

Reply:
The problem is that for normal training you can also provide a folder. In that case the latest zipped file in that folder is taken for evaluation. So we need some variable to distinguish between those.

Reply:
we could also change it similar to the train compare: if you pass a list of models (`tar.gz` files) use compare test, if you pass just a single model (`tar.gz` file) or a directory, do normal testing.

Reply:
Added it to the issue, will tackle this in a separate PR.

Reply:
> we could also change it similar to the train compare: if you pass a list of models (tar.gz files) use compare test, if you pass just a single model (tar.gz file) or a directory, do normal testing.

That sounds good!